ilc/client/getIlcUrlProcessor.js

```diff
@@ -0,0 +1,13 @@
+const UrlProcessor = require('../common/UrlProcessor');
+
+let urlProcessor;
+
+function getIlcUrlProcessor(trailingSlash) {
```

Comment:
Let's make this file more obvious. It should fetch necessary configs from `initIlcConfig` on it's own.

Reply:
Added